bindings/ruby/src/spec/models/cran_package_upload_request_spec.rb

```diff
@@ -35,10 +35,6 @@
   describe 'test attribute "architecture"' do
     it 'should work' do
       # assertion here. ref: https://www.relishapp.com/rspec/rspec-expectations/docs/built-in-matchers
-      # validator = Petstore::EnumTest::EnumAttributeValidator.new('String', ["arm64", "x86_64"])
-      # validator.allowable_values.each do |value|
-      #   expect { @instance.architecture = value }.not_to raise_error
-      # end
```

Comment:
Is this test needed anymore? 🤔

Reply:
Nope, the architecture field is no longer an enum/choice field (due to the incompatibility with the CLI), so the allowable values isn't necessary now.

Reply:
Cool cool, worth just deleting then?